lib/utils/getCacheFile.mjs

```diff
@@ -33,15 +33,9 @@ export default function getCacheFile(cacheFile, cwd) {
 		return join(resolvedCacheFile, `.stylelintcache_${hash(cwd)}`);
 	}
 
-	let fileStats;
+	const fileStats = lstatSync(resolvedCacheFile, { throwIfNoEntry: false });
 
-	try {
-		fileStats = lstatSync(resolvedCacheFile);
-	} catch {
-		fileStats = null;
-	}
-
-	if (looksLikeADirectory || (fileStats && fileStats.isDirectory())) {
+	if (looksLikeADirectory || fileStats?.isDirectory()) {
 		// Return path to provided directory with generated file name.
 		return getCacheFileForDirectory();
```

Comment:
[suggest] I'm not sure why `getCacheFileForDirectory` was created, but it seems we can inline this function. Any thoughts?

```suggestion
		return join(resolvedCacheFile, `.stylelintcache_${hash(cwd)}`);
```

Reply:
I am trying to limit the diff to the minimum.
It's weird, I agree, but it might have a use if exported.
e.g. check l517 of standalone-cache.test.mjs

Reply:
Okay, no problem 👍🏼